cl_manycore/libraries/bsg_manycore_cuda.h

```diff
@@ -21,7 +21,7 @@ extern "C" {
 #define HB_MC_MMIO_TILE_ARGV_PTR_EPA		0x1008	// EPA of pointer to argument list of kernel in tile.
```

Comment:
How about HB_MC_CUDA_TILE_ARGV_PTR_EPA

This is my fault. I meant you to look at the naming scheme there to demonstrate how we've named similar macros, not to copy them verbatim. MMIO is used for addresses that are accessible by AXI. 

I would also recommend adding them to the base offset defined in tile. For example: 

```
define HB_MC_MMIO_CUDA_ARGV_PTR_EPA (HB_MC_TILE_EPA_DMEM_BASE  + 0x08)

Reply:
This means that if we change the EPA base, then these constants will automatically change as well

Reply:
Ahh I see, yeah that makes sense! Done, thanks!